mongoose/mongoose.d.ts

```diff
@@ -276,8 +276,7 @@ declare module "mongoose" {
 
     pre(method: string, fn: HookSyncCallback, errorCb?: HookErrorCallback): Schema;
     pre(method: string, isAsync: boolean, fn: HookAsyncCallback, errorCb?: HookErrorCallback): Schema;
-    post(method: string, fn: HookSyncCallback, errorCb?: HookErrorCallback): Schema;
-    post(method: string, isAsync: boolean, fn: HookAsyncCallback, errorCb?: HookErrorCallback): Schema;
```

Comment:
@RyanCavanaugh 
See http://mongoosejs.com/docs/middleware.html#post

post is different than pre. 

and the change in mongoose is already merged, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9850

I don't know why diff shows it again.